dotCMS/src/main/java/com/dotmarketing/db/commands/DatabaseCommand.java

```diff
@@ -17,6 +17,7 @@
         CONDITIONAL_COLUMN,
         CONDITIONAL_VALUE,
         EXTRA_COLUMNS,
+        DO_NOTHING
```

Comment:
I would doc this such as 

Do nothing on conflict or so

Reply:
Done